### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by cmirdesouza**
Did the following items of #1 
- [x] ADD dependency on anki running for anki-editor to work
- [x] ADD Limitations
  - Only One Way Sync, alternative ankiorg
  - Note Deletions not Synced
  - Note Type Changes?

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -180,6 +181,14 @@ Anki tags before they can be used in Org without any surprise.
    especially those who extend the builtin Anki note editor to
    automatically fill note field content (e.g. ~Add note id~).
 
+*** One Way Sync with Anki
+To sync anki notes and decks to org see ([[https://github.com/orgtre/ankiorg][orgtre/ankiorg]]).
+
+The following items are not synchronized to org:
+ - Deletion of Notes
+ - Deck Changes
```
Comment (orgtre) [inline README.org:189]: Deck changes are actually properly synchronized now!

**Comment by orgtre** [comment]
Thanks! But I think you/we can add some more changes here before merging this.

**Comment by cmirdesouza** [comment]
I was looking for some library to automatically update the documentation using the comments on the elisp or the describe function, what do you think about [org-readme ](https://www.emacswiki.org/emacs/org-readme.el) seems to resolve the problem of outdated documentation of functions and variables.

**Review by cmirdesouza — COMMENTED** [review]
- [x] UPDATE Note Layouts (see changelog.org)
- [x] ADD Explanation on How to Consult Docs of Functions, Commands and Macros

**Comment by orgtre** [comment]
Sure, you can try with org-readme.

**Pull request merged**